index.html

```diff
@@ -73,22 +73,39 @@
 			h1 {
 				font-size: 140%;
 			}
-		</style>
+
+            .embed-container {
+                position: relative;
+                height: 0;
+                overflow: hidden;
+            }
+
+            .ratio-audience {
+                padding-bottom: 48.33333333333333%;
+            }
+
+            .embed-container img {
+                position: absolute;
+                top: 0;
+                left: 0;
+                width:100%;
+            }
+        </style>
 		
 		<link rel="stylesheet" type="text/css" href="css/animate.css"/>
 	</head>
-	<body onload="animate()">
+	<body>
```

Comment:
Mmh… I definitely understand why you'd do that, but don't let the current syntactical ugliness (no proper handler attachment) hide the expected behavior. The animation may now start on images that have not loaded yet, and we cannot predict the time at which resources will be available.